### PR TITLE
feat: add a distinctions in sources and API for protected data sources

### LINKED ISSUE #1095: [Espace agent] Sources & API
- afficher une distinction entre les sources agents et les autres

### PR DESCRIPTION
**Pull request by XavierJp**
- Creates a distinction between data sources (protected vs open)
- Revamp changelog page with espace agent vs open distinction
- Add missing sources (qualifelec, qualibat and opbibi)

closes #1095

### CODE REVIEW

**Comment by XavierJp** [comment]
image

image

File: data/changelog.yml [hunk 1]
```diff
@@ -1,89 +1,128 @@
...
+    Nouvelle source de données et nouveau filtre : les Entreprises Sociales Inclusives
 - date: 24/02/2024
-  title: Ajout du filtre géographique EPCI
+  body: |
+    Ajout du filtre géographique EPCI
 - date: 14/02/2024
-  title: 'Nouvelle source de donnée, [l’Annuaire du service public](https://lannuaire.service-public.fr/) :'
-  description: |
+  body: |
+    Nouvelle source de donnée, [l’Annuaire du service public](https://lannuaire.service-public.fr/) :
```
Comment (HAEKADI) [inline data/changelog.yml:14]:
```suggestion
    Nouvelle source de données, [l’Annuaire du service public](https://lannuaire.service-public.fr/) :
```

File: data/changelog.yml [hunk 2]
```diff
@@ -1,89 +1,128 @@
...
-  title: 'L’[Insee](/administration/insee) a mis en place une nouvelle politique de non-diffusion des données des entrepreneurs individuels :'
-  description: |
+  body: |
+    L’[Insee](/administration/insee) a mis en place une nouvelle politique de non-diffusion des données des entrepreneurs individuels :
+
     Désormais, seuls l’adresse et les nom et prénom de l’entrepreneur ne sont pas diffusés. Le reste des informations sont publiques ([en savoir plus](https://www.insee.fr/fr/information/6683782)).
 - date: 09/02/2023
-  title: "Enrichissement de la section association : agréments administratif, éligibilité CEC, information de contact (email, site web, tel), association d'intérêt public et régime d’association."
+  body: |
+    "Enrichissement de la section association : agréments administratif, éligibilité CEC, information de contact (email, site web, tel), association d'intérêt public et régime d’association."
```
Comment (HAEKADI) [inline data/changelog.yml:63]:
```suggestion
    "Enrichissement de la section association : agréments administratifs, éligibilité CEC, informations de contact (email, site web, tel), association d'intérêt public et régime d’association."
```

File: models/administrations/data-to-modify.ts [hunk 3]
```diff
@@ -27,6 +27,11 @@ const loadDataToModify = () => {
   );
 };
 
+/**
+ * List the different data tht can be modifiedby users - useful in FAQ
```
Comment (HAEKADI) [inline models/administrations/data-to-modify.ts:31]:
```suggestion
 * List the different data that can be modified by users - useful in FAQ
```

**Review by HAEKADI — APPROVED** [review]

**Pull request merged**